src/components/AddReminder/AddReminder.tsx

```diff
@@ -48,14 +27,21 @@ const AddReminder = (props: Props) => {
         Add Reminder
         <IconButton
           aria-label="Close"
-          className={classes.closeButton}
+          sx={{ position: "absolute", right: "10px", top: "10px" }}
```

Comment:
With the latest version of @mui classNames are no longer used and the way to extend styles is by using sx property and passing an emotion compatible object.